Fix enchant command feedback messages

Machine-Maker opened the pull request:
Fixes https://github.com/PaperMC/Folia/issues/12

This issue probably exists in other places where "feedback messages" are sent in commands, but I just wanted to see if this was an acceptable solution to the issue. 

Amaranth also suggested you could keep some atomic int counters that incremented/decremented appropriately so in the last of the scheduled entity tasks, the messages could be sent there so that's an alternative solution.

kashike commented on patches/server/0004-Threaded-Regions.patch:
```diff
@@ -12631,31 +12635,67 @@ index e639c0ec642910e66b1d68ae0b9208ef58d91fce..9ad71bda2f7498ad6e0853a1070c5be2
 +                    entity.getBukkitEntity().taskScheduler.schedule((LivingEntity nmsEntity) -> {
 +                        try {
-+                            LivingEntity livingEntity = (LivingEntity)nmsEntity;
++                            LivingEntity livingEntity = (LivingEntity) nmsEntity;
```
tsk

Spottedleaf commented:
This isn't acceptable due to the usage of completablefutures. The exception handling is broken, as you've swallowed every other exception that isn't CommandSyntaxException. The proper handling is to re-throw those exceptions up the stack, but that isn't possible with completablefutures.

Machine-Maker commented:
Ok, I removed the use of completable futures and moved over to sending the message when the last of the targets is processed, either inside the scheduled task or not (if its an invalid entity).
